src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml

```diff
@@ -0,0 +1,27 @@
+<UserControl
+    x:Class="ArcGIS.WinUI.Samples.AddVectorTiledLayerFromCustomStyle.AddVectorTiledLayerFromCustomStyle"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <TextBlock
+                    Margin="0,0,0,5"
+                    HorizontalAlignment="Center"
+                    FontWeight="SemiBold"
+                    Text="Select style:"
+                    TextWrapping="Wrap" />
+                <ComboBox x:Name="StyleChooser" SelectionChanged="StyleChooserSelectionChanged">
```

Comment:
Nitpick: the `ComboBox` defaults to an unselected value but the basemap shows the `Default` option. I'd set the value of the `ComboBox` to default on initialize so the user knows what they are viewing when the sample loads. The current functionality is slightly confusing as the user can select `Default` and trigger a reload when the initial state is already `Default`. I'd also add a minimum width to the `ComboBox`.